contractcourt/commit_sweep_resolver.go

```diff
@@ -221,8 +221,8 @@ func (c *commitSweepResolver) Resolve() (ContractResolver, error) {
 		// If we have both a csv and cltv lock, we'll need to look at
 		// both and see which expires later.
 		case c.commitResolution.MaturityDelay > 0 && c.hasCLTV():
-			c.log.Debugf("waiting for CSV and CLTV lock to expire "+
-				"at height %v", unlockHeight)
+			c.log.Debugf("waiting for CSV (%v) and CLTV (%v) lock "+
+				"to expire", unlockHeight, c.leaseExpiry)
```

Comment:
here we only wait on the `unlockHeight` tho

Reply:
hmm really:

```
waitHeight = uint32(math.Max(
				float64(unlockHeight-1), float64(c.leaseExpiry),
			))
```

thats why I was adding both log files, was hard for me to reason about the LeaseScript tests without it.